script/index.js

```diff
@@ -4,7 +4,7 @@ import GramSchmidt from "./module/GramSchmidt.js";
 
 const ferramentas = new FerramentasAux();
 const baseEspaco = ferramentas.init();
-const base = ferramentas.povoarBase(baseEspaco[1]);
+const base = ferramentas.povoarBase();
```

Comment:
acho que aqui, sem passar o parâmetro e não usando mais aquela variável this.base sendo declarada dentro dela não vai funcionar, vamo deixar a linha antiga mesmo

Reply:
Certo, corrigido no novo commit.